securedrop/tests/functional/factories.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
```

Comment:
This file contains two factories, which follow the API convention of the `factoryboy` library (in case this library is ever added to the unit tests dependencies). The factories can later be used in more code locations.